nginx.conf

```diff
@@ -37,9 +37,6 @@ http {
             index  index.html index.htm;
             add_header 'Access-Control-Allow-Origin' '*' always;
             add_header 'Access-Control-Expose-Headers' 'content-length' always;
-            add_header 'Cross-Origin-Opener-Policy' 'same-origin' always;
-            add_header 'Cross-Origin-Embedder-Policy' 'require-corp' always;
-            add_header 'Cross-Origin-Resource-Policy' 'cross-origin' always;
```

Comment:
I think we should keep these for reference, the purpose of this work is to require the requirement for this config, but the config is still fine